test/inductor/test_aot_inductor.py

```diff
@@ -1177,6 +1214,7 @@ class AOTInductorTestABICompatibleCpu(TestCase):
         # TODO: test_freezing_abi_compatible_cpu somehow fails on CI but not locally,
         #   NotImplementedError: Cannot access storage of OpaqueTensorImpl
         "test_freezing": TestFailure(("abi_compatible_cpu",), is_skip=True),
+        "test_missing_cubin": TestFailure(("abi_compatible_cpu",)),
```

Comment:
What will happen if we run cpu tests? It is true that the main purpose is to store/load cubin in different stage, but in theory it should still work fine if we run the exact CPU test?

Reply:
We need to add abi_compatible support for convolution, and the issue is more than adding a missing shim interface. I will add a comment, and come back to fix it later.